lib/copy_clover_offset.cu

```diff
@@ -0,0 +1,83 @@
+#include <clover_field_order.h>
+#include <instantiate.h>
+#include <kernels/copy_field_offset.cuh>
+
+namespace quda
+{
+
+  using namespace clover;
+
+  template <typename Float> struct CopyCloverOffset {
+    CopyCloverOffset(CloverField &out, const CloverField &in, CommKey offset, bool inverse)
+    {
+      constexpr int length = 72;
```

Comment:
Same comment as above if there's a way to query this `72` from `CloverField` or whatnot

Reply:
Addressed above by @maddyscientist , resolved.